app/uk/gov/hmrc/helptosavereminder/config/AppConfig.scala

```diff
@@ -17,16 +17,32 @@
 package uk.gov.hmrc.helptosavereminder.config
 
 import javax.inject.{Inject, Singleton}
-import play.api.Configuration
+import play.api.{Configuration, Environment, Mode}
 import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
 @Singleton
-class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig) {
+class AppConfig @Inject()(val config: Configuration, val environment: Environment, val servicesConfig: ServicesConfig) {
+
+  protected def mode: Mode = environment.mode
```

Comment:
Why introduce Mode, I didn't think we used that anywhere anymore?

Reply:
Yes can do